lineman/app/services/lmo_url_service.coffee

```diff
@@ -12,6 +12,9 @@ angular.module('loomioApp').factory 'LmoUrlService', (AppConfig) ->
     routePath: (route) ->
       "/".concat(route).replace('//', '/')
 
+    membershipRequest: (mr, params = {}, options = {}) ->
+      @buildModelRoute('g', mr.group().key, 'membership_requests', params, options)
```

Comment:
@gdpelican - checking if correct use of this method.. it works but is it right to put the path there?